catalogue_graph/src/models/catalogue_concept.py

```diff
@@ -52,31 +52,29 @@ def standardise_label(label: str | None) -> str | None:
     return capitalised.replace("--", " - ")
 
 
-def get_priority_source_concept_value(
-    concept_node: dict | None, source_concept_nodes: list[dict], key: str
-) -> tuple[Any, str | None]:
+def get_priority_label(
```

Comment:
This is nice, reduces the scope of the function, makes it clearer what it's doing.